app/uk/gov/hmrc/teamsandrepositories/config/JenkinsConfig.scala

```diff
@@ -25,4 +25,8 @@ class JenkinsConfig @Inject()(config: Configuration) {
   lazy val token: String = config.get[String]("jenkins.token")
 
   lazy val baseUrl: String = config.get[String]("jenkins.url")
+
+  lazy val queueThrottleInSeconds: Int = config.get[Int]("jenkins.queue.throttle.seconds")
```

Comment:
Can we configure these as durations rather than adding the expected units to the key name.
`jenkins.queue.throttle=10.seconds`